less/variables.less

```diff
@@ -13,7 +13,7 @@
 @C2C-orange: #FFAA45;
 @bg-color1 : #FBFBFB;
 @page-header-height: 48px;
-@menu-width: 250px;
+@menu-width: 190px;
```

Comment:
OK but then please recenter the logo:
<img width="253" alt="capture d ecran 2017-11-28 a 21 14 05" src="https://user-images.githubusercontent.com/1192331/33342065-1947d3dc-d481-11e7-9f74-519db4334c7a.png">

Reply:
Done !